sqlglot/dialects/clickhouse.py

```diff
@@ -66,6 +67,11 @@ class Parser(parser.Parser):
             "XOR": lambda args: exp.Xor(expressions=args),
         }
 
+        TYPE_TOKENS = {
+            *parser.Parser.TYPE_TOKENS,
+            TokenType.FIXEDSTRING,
```

Comment:
this should be in the base parser, not clickhouse

Reply:
This is a very clickhouse-specific feature - I deliberately didn't include it in the base parser because it's not really relevant to anyone else. Is there an other reason I missed, besides "extending" the sqlglot dialect?

Reply:
TokenTypes are global, so there's no reason not to put it in base, i've already made it clickhouse specific by puting the string -> token mapping only in clickhouse

Reply:
Got it, makes sense I guess because we'll also avoid creating a copy of `TYPE_TOKENS` in this way.

Making in the fix in a bit.

Reply:
Done